docs/openapi/paths/password.yml

```diff
@@ -0,0 +1,35 @@
+---
+post:
+  tags:
+    - Account
```

Comment:
While I agree with putting the Registration in Account tag, the Forgot Password should be in `Authentication`, no?
As it allows users who have forgotten their password to reset it and regain access to their account, and this is generally implemented in the Login page 🤔

Reply:
Fixed in https://github.com/nimblehq/nimble-survey-web/pull/96/commits/c510365a35513f5cda2e2fd5056645cd83a87fe9